README.md

```diff
@@ -1 +1,175 @@
-Work in progress (WIP)
\ No newline at end of file
+# cms-ars-5.0-k8s-cluster-stig-overlay
+
+InSpec profile to validate the secure configuration of Kubernetes cluster against [DISA's](https://public.cyber.mil/stigs/) Kubernetes Secure Technical Implementation Guide (STIG) Version 1 Release 1 tailored for CMS ARS 5.0.
+
```

Comment:
Should a paragraph like this be inserted before the **Getting Started** section?

> The baseline includes tests that must be executed against the Kubernetes API and tests that must be executed against the underlying nodes over SSH. This profile handles only the cluster tests, and is intended to be run alongside the [CMS ARS 5.0 EKS CIS Node Overlay profile](https://github.com/CMS-Enterprise/cms-ars-5.0-eks-cis-node-overlay).

Of course, the link to the overlay will be replaced with CMS ARS Kubernetes Node Overlay link.

Reply:
yes. done.